### PR TITLE
[FAD-169] Backer Withdraw

### PR DESCRIPTION
**Pull request by sachasmart-weavik**
### Description

> Backer withdraw from a pool through their pool nfts. 

[Testnet Link](https://testnet.aurorascan.dev/tx/0xe859b98799b353e1f66c22ab1784856aa54f6b2593bd87a3ebdc30b6e8b06f75)
### Checklist

- [ ] `CHANGELOG` has been updated (_if appropriate_)
- [ ] Environment variables updated (_if appropriate_)

### CODE REVIEW

File: packages/client3/pages/backer/pool/[address]/backer-withdraw-cta.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      totalTokenValue +=
+        BigNumber.from(
+          backerTokenInformation[i].principalRedeemable
+        ).toNumber() +
+        BigNumber.from(backerTokenInformation[i].interestRedeemable).toNumber();
+      dropDownOptions.push(backerTokenInformation[i].id);
+    }
+    setTotalTokenValue(totalTokenValue);
+    setOptions(dropDownOptions);
+  }, [backerTokenInformation]);
```
Comment (sachasmart-weavik) [inline packages/client3/pages/backer/pool/[address]/backer-withdraw-cta.tsx:52]: @v1n33th Thoughts on this...

File: packages/client3/components/pool/pool-information/pool-information.tsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ import {
 import { openVerificationModal } from "@/lib/state/actions";
 import { toastTransaction } from "@/lib/toast";
 import { isSmartContract, useWallet } from "@/lib/wallet";
+import { BackerWithdrawCard } from "@/pages/backer/pool/[address]/backer-withdraw-card";
```
Comment (v1n33th) [inline packages/client3/components/pool/pool-information/pool-information.tsx:35]: Not needed i believe

File: packages/client3/pages/backer/pool/[address]/backer-withdraw-card.tsx [hunk 3]
```diff
@@ -1,96 +1,80 @@
...
-        </Form>
-      </div>
+  const onBackerWithdraw = async (tokenId: string, amount: string) => {
+    poolContract.withdraw(
+      BigNumber.from(tokenId).toNumber(),
+      BigNumber.from(amount).toNumber()
+    );
+  };
+
+  /** !Note: TBH, this doesn't make sense to me at all. Conceivably this could make sense if the pool's "value" were increasing over time.
```
Comment (v1n33th) [inline packages/client3/pages/backer/pool/[address]/backer-withdraw-card.tsx:45]: 😅

File: packages/client3/pages/backer/pool/[address]/backer-withdraw-cta.tsx (hunk 1 above)
Comment (v1n33th) [reply packages/client3/pages/backer/pool/[address]/backer-withdraw-cta.tsx:52]: Looks good to me

File: packages/client3/pages/backer/pool/[address]/backer-withdraw-cta.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+          Available Funds
+        </BodyText>
+        <Display level={2} className="text-accent-2">
+          {formatCrypto({
+            token: SupportedCrypto.Usdc,
+            amount: BigNumber.from(totalTokenValue),
+          })}
+        </Display>
+
+        {backerTokenInformation.length > 1 && (
```
Comment (v1n33th) [inline packages/client3/pages/backer/pool/[address]/backer-withdraw-cta.tsx:80]: Just to understand, Any reasoning to use length>1 instead of >0

**Review by v1n33th — APPROVED** [review]
Awesome...Looks good to me

**Pull request merged**